extensions/wikia/Development/ExternalStoreDBFetchBlobHook.php

```diff
@@ -91,7 +107,7 @@ function ExternalStoreDBFetchBlobHook( $cluster, $id, $itemID, &$ret ) {
 		}
 	}
 	else {
-		wfDebug( __METHOD__ . ": malformed response from API call\n" );
+		wfDebug( sprintf( "%s: malformed response from API call (%s)\n", __METHOD__, json_encode( func_get_args() ) ) );
```

Comment:
`__METHOD__` is the second argument of `sprintf`. I like __METHOD__ as much as you do and not going to get rid of it ;)